### PR TITLE
Edit reset service password section

### LINKED ISSUE #866: [Docs RFC] Add instructions to set authentication type for Cloud service password to MD5
# Describe change in content, appearance, or functionality

With self-hosted TimescaleDB, you can edit the `postgresql.conf` file to change password encryption type from SCRAM-SHA-256 to MD5. You can do the same thing on Cloud via the UI, but that is not documented yet.

# Subject matter expert (SME)

@tylerfontaine @clarkbw 

# Deadline

[When does this need to be addressed]

# Any further info

Select the service. Go to Operations -> Reset Password.
It will ask for your Timescale Cloud Account password to proceed.
Then you can set your new password and select MD5 instead of SCRAM-SHA
image

### PR DESCRIPTION
**Pull request by charislam**
# Description

- Correct Operations/Password section to clarify that it resets the service password, not the Cloud password
- Add mention that you can change from SCRAM to MD5 authentication

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #866

### CODE REVIEW

**Review by clarkbw — APPROVED** [review]
Looks great! I can't seem to test the search in the preview URL. Hopefully when this merges it'll update and searching for `SCRAM` will yield this section. 🙇🏼

File: cloud/operations.md [hunk 1]
```diff
@@ -36,10 +36,16 @@ complete the transition before you start forking.
...
-Use this section to reset the password you use to log in to your Timescale Cloud
-account. You must have access to the current password to do this. If you have
-forgotten your password, you can reset it from the login screen instead.
+### Reset service password
+You can reset your service password from the `Operations` dashboard. This is the
+password you use to connect to your database, not the password to your Timescale
+Cloud account. To reset your Timescale Cloud password, navigate to the `Account`
+page. 
+
+When you reset your service password, you are first prompted for your Timescale
```
Comment (Loquacity) [inline cloud/operations.md:45]:
```suggestion
When you reset your service password, you are prompted for your Timescale
```

File: cloud/operations.md [hunk 2]
```diff
@@ -36,10 +36,16 @@ complete the transition before you start forking.
...
+Cloud password. Once you authenticate, you can create a new service password,
```
Comment (Loquacity) [inline cloud/operations.md:46]:
```suggestion
Cloud password. When you have authenticated, you can create a new service password,
```

File: cloud/operations.md [hunk 3]
```diff
@@ -36,10 +36,16 @@ complete the transition before you start forking.
...
+ask Timescale Cloud to auto-generate a password, or switch your authentication
+type between SCRAM and MD5.
```
Comment (Loquacity) [inline cloud/operations.md:48]: Can we add a short description of SCRAM and MD5 here?

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: cloud/operations.md (hunk 3 above)
Comment (clarkbw) [reply cloud/operations.md:48]: SCRAM = Salted Challenge Response Authentication Mechanism - https://en.wikipedia.org/wiki/Salted_Challenge_Response_Authentication_Mechanism
MD5 = Message Digest Algorithm  - https://en.wikipedia.org/wiki/MD5

Comment (charislam) [reply cloud/operations.md:48]:
```suggestion
type between SCRAM and MD5.

SCRAM (Salted Challenge Response Authentication Mechanism) and MD5 (Message Digest
Algorithm 5) are cryptographic authentication mechanisms. Timescale Cloud uses SCRAM
by default. It is more secure and strongly recommended. The MD5 option is provided
for compatibility with older clients.
```

How about this?

Comment (clarkbw) [reply cloud/operations.md:48]: 💯

**Review by Loquacity — APPROVED** [review]

**Review by Rajakavitha1 — APPROVED** [review]

**Pull request merged**